ovmf-vars-generator

```diff
@@ -30,7 +30,7 @@ def strip_special(line):
 def generate_qemu_cmd(args, readonly, *extra_args):
     is_x86 = True
     arch_args = []
-    if os.path.basename(args.qemu_binary) == 'qemu-system-aarch64':
+    if os.path.basename(args.qemu_binary) == 'qemu-system-aarch64' or args.arch == 'aarch64':
```

Comment:
Personally I'd prefer we not make any assumptions about what the qemu_binary path means and just blindly use whatever they pass in. I think `--arch` should be the one and only way to determine what arguments are passed to that binary. If they user doesn't specify, we can then pick a reasonable default qemu_binary based on the arch (where arch is x86_64 by default).